Jasmine/Common/ViewControllers/GameElements/SquareGridViewController.swift

```diff
@@ -51,18 +55,16 @@ class SquareGridViewController: UIViewController {
     ///   - initialData: initial set of data to be displayed in this view.
     ///   - numRows: maximum number of rows to be displayed in this grid.
     ///   - numCols: maximum number of columns to be displayed in this grid.
-    func segueWith(_ initialData: [Coordinate: String], numRows: Int, numCols: Int) {
```

Comment:
No. some users of this class do not have GridData. For example, Riwu's upcoming tiles.